main_2024.py

```diff
@@ -212,6 +216,9 @@ def main() -> int:
             FLIGHT_INTERFACE_TIMEOUT,
             FLIGHT_INTERFACE_BAUD_RATE,
             FLIGHT_INTERFACE_WORKER_PERIOD,
+            flight_interface_to_data_merge_queue,
+            flight_interface_decision_queue,
+            controller,
```

Comment:
flight_interface_to_data_merge_queue should go in the output_queues arg, and flight_interface_decision_queue should go in the input_queues arg